airflow-postgres-app-services/azuredeploy.json

```diff
@@ -0,0 +1,227 @@
+            "type": "string",
+            "allowedValues": [
+                "9.5",
+                "9.6",
+                "10",
+                "11"
+            ],
+            "defaultValue": "11",
+            "metadata": {
+                "description": "PostgreSQL version"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        }
+    },
+    "variables": {
+        "databaseName": "[concat(parameters('airflowWebAppName'), 'database')]",
+        "serverName": "[concat(parameters('airflowWebAppName'), 'pgserver')]",
+        "hostingPlanName": "[concat(parameters('airflowWebAppName'), 'serviceplan')]"
+    },
+    "resources": [
+        {
+            "apiVersion": "2018-11-01",
+            "type": "Microsoft.Web/serverfarms",
+            "name": "[variables('hostingPlanName')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "name": "[variables('hostingPlanName')]",
+                "workerSize": "1",
+                "reserved": true,
+                "numberOfWorkers": 1
+            },
+            "sku": {
+                "Tier": "Standard",
+                "Name": "S1"
+            }
+        },
+        {
+            "apiVersion": "2018-11-01",
+            "type": "Microsoft.Web/sites",
+            "name": "[parameters('airflowWebAppName')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "serverFarmId": "[variables('hostingPlanName')]",
+                "name": "[parameters('airflowWebAppName')]",
+                "siteConfig": {
+                    "linuxFxVersion": "DOCKER|puckel/docker-airflow:latest",
+                    "requestTracingEnabled": true,
+                    "httpLoggingEnabled": true,
+                    "detailedErrorLoggingEnabled": true,
+                    "autoHealEnabled": true
+                },
+                "httpsOnly": true
+            },
+            "resources": [
+                {
+                    "apiVersion": "2019-08-01",
+                    "type": "config",
+                    "name": "appsettings",
+                    "properties": {
+                        "AIRFLOW__CORE__SQL_ALCHEMY_CONN": "[concat('postgresql://', parameters('administratorLogin'), ':', parameters('administratorLoginPassword'),'@',reference(resourceId('Microsoft.DBforPostgreSQL/servers',variables('serverName'))).fullyQualifiedDomainName,':5432/',variables('databaseName'))]",
+                        "AIRFLOW__CORE__SQL_ALCHEMY_POOL_SIZE": 20,
+                        "AIRFLOW__CORE__LOAD_EXAMPLES": "true",
+                        "AIRFLOW__CORE__EXECUTOR": "LocalExecutor",
+                        "WEBSITES_ENABLE_APP_SERVICE_STORAGE": "true"
+                    },
+                    "tags": {
```

Comment:
looks like a lot of the resources need to be sorted, see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

Reply:
Updated